src/config/createFilesConfig.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
If you made this a `.ts` file you could do the `path` `join` in here and export it? might make the less verbose `renderFiles`